tools/devices/fe310/fe310-g000.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<modm version="0.3.2">
+  <device platform="fe310" family="g0" name="00">
+    <naming-schema>{platform}-{family}{name}</naming-schema>
+
+    <!-- Core -->
+    <attribute-core value="e31"/>
+
+    <driver name="core" type="e31">
+      <memory name="flash" access="rx" start="0x20000000" size="16384"/>
+      <memory name="dtim" access="rwx" start="0x80000000" size="16"/>
+    </driver>
+
+    <driver name="clock" type="e31"/>
+
+    <driver name="gpio" type="e31">
+      <port name="A" base="0x10012000"/>
```

Comment:
Should contain this information
![Screenshot 2019-09-20 at 00 41 21](https://user-images.githubusercontent.com/163066/65286380-a76ef480-db3f-11e9-9079-86c6e189313f.png)